controllers/grid/settings/sections/form/PKPSectionForm.inc.php

```diff
@@ -49,33 +49,6 @@ function __construct($request, $template, $sectionId = null) {
 		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_SUBMISSION, LOCALE_COMPONENT_PKP_MANAGER);
 	}
 
-	/**
-	 * @copydoc Form::validate()
-	 */
-	function validate($callHooks = true) {
-		// Validate if it can be inactive
-		if ($this->getData('isInactive')) {
-			$request = Application::get()->getRequest();
-			$journal = $request->getJournal();
-			$sectionId = $this->getSectionId();
-
-			$sectionDao = DAORegistry::getDAO('SectionDAO'); /* @var $sectionDao SectionDAO */
-			$sectionsIterator = $sectionDao->getByContextId($journal->getId());
-			$activeSectionsCount = 0;
-			while ($section = $sectionsIterator->next()) {
-				if (!$section->getIsInactive() && ($sectionId != $section->getId())) {
-					$activeSectionsCount++;
-				}
-			} 
-			if ($activeSectionsCount < 1 && $this->getData('isInactive')) {
-				$this->addError('isInactive', __('manager.sections.confirmDeactivateSection.error'));
-				$this->addErrorField('isInactive');
-			}
-		}
-
-		return parent::validate($callHooks);
-	}
-
```

Comment:
Good catch on this! It looks like the same change needs to be applied to `PKPSubmissionStep1Form::validate()`. It needs to be moved into `SubmissionStep1Form` for each app.